utils/canutils/Makefile

```diff
@@ -8,13 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=canutils
-PKG_VERSION:=2023.03
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=v$(PKG_VERSION)
```

Comment:
Hey, I'm looking at this and from the commit description I didn't find out why you actually removed ``PKG_VERSION``? . Now you've completely changed the package versioning and that's not exactly good. If you decided to remove PKG_VERSION, there must be ``PKG_SOURCE_DATE``. :) In ideal case.

I think switching to 2025.01 is just fine.